src/main/resources/templates/agenda.html

```diff
@@ -59,6 +59,19 @@ <h2>Stemagenda</h2>
 </html>
 
 <script>
+  $.tablesorter.addParser({
+  // use a unique id
+  id: 'dutchdateparser',
+  is: function(s, table, cell, $cell) {
+    return false;
+  },
+  format: function(s, table, cell, cellIndex) {
+    return new Date(s).getTime();
+  },
+  // flag for filter widget (true = ALWAYS search parsed values; false = search cell text)
+  parsed: false,
+  type: 'numeric'
+});
   $("#agenda").tablesorter()
   $("#vote-agenda").tablesorter()
```

Comment:
toukie.html also contains a table with incorrectly sorted dates at the moment, if you want to you could also add this sorter there.